src/auth0_server_python/auth_server/server_client.py

```diff
@@ -611,7 +639,9 @@ async def get_access_token(self, store_options: Optional[dict[str, Any]] = None)
         # Get new token with refresh token
         try:
             token_endpoint_response = await self.get_token_by_refresh_token({
-                "refresh_token": state_data_dict["refresh_token"]
+                "refresh_token": state_data_dict["refresh_token"],
+                "audience": audience,
+                "scope": scope
```

Comment:
Scope seems to be unconditionally added, meaning we introduce Access Token descoping for non MRRT use-cases, which seems to be not intentional.